.github/workflows/jirabot-merge-process.yml

```diff
@@ -0,0 +1,80 @@
+# JiraBot - Process Merge (Privileged)  
```

Comment:
I am not sure I see the benefit in splitting the jirabots into "pre" and "post" steps. The post step will still require the ability to write to the pull request. How does this improve security beyond keeping the the event as pull_request_target and limiting permissions to contents: read, pull-requests: write?